### DISCUSSION TITLE
A better place for discussion

### DISCUSSION THREAD

**Discussion by qoobes** [CopilotC-Nvim/CopilotChat.nvim#121] [General]
Let's make something like a discord server @jellydn. This will make it quite a bit easier to coordinate, organise, and attract more contributors.

**Comment by cjoke** [comment]
Hi qoobes, 
There are a discord channel https://github.com/CopilotC-Nvim/CopilotChat.nvim/blob/main/README.md#copilot-chat-for-neovim read the last line in this section of README.md document.

**Reply by qoobes** [reply]
I don't see how I missed that....

**Reply by cjoke** [reply]
Happens :)
